### PR TITLE
feat: add get chunk size

### PR DESCRIPTION
**Pull request by ruleeeer**

### CODE REVIEW

File: controllers/packageSize.js [hunk 1]
```diff
@@ -1,70 +1,86 @@
...
+  myHeaders.append('Host', 'bundlephobia.com');
+  myHeaders.append('Connection', 'keep-alive');
+  const requestOptions = {
+    method: 'GET',
+    headers: myHeaders,
+    redirect: 'follow',
+  };
+
+  console.time('fetchTime');
+  const response = await fetch(`https://bundlephobia.com/api/size?package=${name}&record=true`, requestOptions);
```
Comment (flyfishzy) [inline controllers/packageSize.js:18]: keep version param

File: controllers/packageSize.js [hunk 2]
```diff
@@ -1,70 +1,86 @@
...
-    const packages = item.ProjectPackage.dataValues;
-    const packageInfo = await syncSinglePackageSize(packages.package);
-    if (packageInfo.erorr) {
-      debug.log('package name: ', packages.package, ' there is no data with package size present');
-      continue;
-    }
+  const packageList = await sequelize.query(query, { type: sequelize.QueryTypes.SELECT });
+
+  for (const { package: packageName } of packageList) {
+    console.debug(`get packageName:${packageName} size data`);
```
Comment (flyfishzy) [inline controllers/packageSize.js:73]: use debug.error()

File: util/database.js [hunk 3]
```diff
@@ -5,9 +5,6 @@ const sequelize = new Sequelize(
   process.env.DATABASE_URL,
   {
     dialectOptions: {
-      ssl: {
```
Comment (flyfishzy) [inline util/database.js:8]: keep ssl

File: controllers/packageSize.js [hunk 4]
```diff
@@ -1,70 +1,86 @@
...
   }
 }
 
 export async function syncPackageSizeHandler(req, res) {
   try {
     if (req.body.name) {
-      syncSinglePackageSize(req.body.name, req.body.version);
+      await syncSinglePackageSize(req.body.name, req.body.version);
     }
     const result = await syncAllPackageSize();
```
Comment (flyfishzy) [inline controllers/packageSize.js:85]: need a else

File: controllers/packageSize.js (hunk 1 above)
Comment (ruleeeer) [reply controllers/packageSize.js]: done

File: util/database.js (hunk 3 above)
Comment (ruleeeer) [reply util/database.js:8]: done

File: controllers/packageSize.js (hunk 4 above)
Comment (ruleeeer) [reply controllers/packageSize.js]: done

**Pull request merged**